packages/eslint-config-kibana/typescript.js

```diff
@@ -17,21 +14,9 @@ module.exports = {
       plugins: [
         '@typescript-eslint',
         'ban',
-        'import',
         'prefer-object-spread',
       ],
 
-      settings: {
-        'import/resolver': {
-          node: {
-            extensions: ['.mjs', '.js', '.json', '.ts', '.tsx'],
-          },
-        },
-        react: {
-          version: semver.valid(semver.coerce(PKG.dependencies.react)),
-        },
```

Comment:
This was redundant as already present in `packages/eslint-config-kibana/react.js`